main.go

```diff
@@ -306,6 +307,12 @@ func (s *slackOutput) SendBatch(batch [][]byte, tag string) error {
 		Text:     strings.Join(messages, "\n"),
 		slackTag: msgTag,
 	}
+
+	// Notification-service alerts should be passed in raw, without slack doing link/ID parsing.
+	if message.Username == "notice" {
```

Comment:
Can we make this even more strict, i.e. also check if `msgTag.Channel ==  "#notification-catcher"`

Reply:
Done.

Reply:
Thanks!